### PR TITLE
"Freed the blocks from the evil CompatLayer!"

### PR DESCRIPTION
**Pull request by azuraerae**
- Fixed various doubles in translation file.

- Gave all the blocks proper registryNames (i.e copper_ore) and changed en_US.lang and all textures/blockstates accordingly.

- Made two Block constructors BlockBase (and ItemBase) and OreBase with according functions. (Drop chance, dropped item for ease of access.)

- Switched the block registration method from the Compatibility1_12_2 layer to the default e.getRegistry() registration method. (Switching back to Forge's default.) -> Made a proper init package, with Blocks (and Items) classes to make proper use of the @SubscribeEvent flags. -> Replaced all Block calls from the Ores class to the new Blocks class.

-> Planning on doing the same for Items (and eventually getting rid of the Ores class alltogether).

- Raeilgun (Azurae)

### CODE REVIEW

File: src/main/java/com/paneedah/mwc/RecipeManager.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/RecipeManager.java:18]: Need to revert that this isn't the Minecraft id this is just the variable should follow Java convention

File: src/main/java/com/paneedah/mwc/bases/BlockBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/BlockBase.java:19]: Missing the space

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:15]: Missing space before bracket

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:12]: Missing space between class init and variables

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:15]: Missing space between method and variable

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:36]: Single line if block should not have brackets

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:45]: Single line if block should not have brackets

**Comment by Paneedah** [comment]
All good on my end! Waiting for @JustDesoroxxx to review before merge.

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:35]: this should probably be close together

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:36]: Missing space between two different actions

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:40]: Should be with the others

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:58]: `e` should be `blockRegistryEvent`

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:70]: `e` should be `itemRegistryEvent`

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:79]: `e` should be `modelRegistryEvent`

File: src/main/java/com/paneedah/mwc/init/Items.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/init/Items.java:9]: Missing space between class init and actuall stuff

File: src/main/java/com/paneedah/mwc/bases/ItemBase.java
Comment (JustDesoroxxx) [inline src/main/java/com/paneedah/mwc/bases/ItemBase.java:6]: Missing space between class init and actuall stuff

**Review by JustDesoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/mwc/RecipeManager.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/RecipeManager.java:18]: ✅

File: src/main/java/com/paneedah/mwc/bases/BlockBase.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/bases/BlockBase.java:19]: ✅

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/bases/OreBase.java:15]: ✅

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/bases/OreBase.java:12]: ✅

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/bases/OreBase.java:15]: ✅

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/init/Blocks.java:58]: ✅

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/init/Blocks.java:70]: ✅

File: src/main/java/com/paneedah/mwc/init/Items.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/init/Items.java:9]: ✅

File: src/main/java/com/paneedah/mwc/bases/ItemBase.java
Comment (azuraerae) [reply src/main/java/com/paneedah/mwc/bases/ItemBase.java:6]: ✅

File: src/main/java/com/paneedah/mwc/bases/BlockBase.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/bases/BlockBase.java:12]: Missing space between class init and actuall stuff

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:37]: Single line if block should not have brackets

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:46]: Single line if block should not have brackets

File: src/main/java/com/paneedah/mwc/init/Items.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/init/Items.java:11]: Missing space between variable and method

File: src/main/java/com/paneedah/mwc/init/Blocks.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/init/Blocks.java:77]: Single line if block should not have brackets

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/mwc/bases/OreBase.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/bases/OreBase.java:16]: Missing space between variables and method

File: src/main/java/com/paneedah/mwc/init/Items.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/init/Items.java:11]: Missing space between variables and method

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/mwc/bases/BlockBase.java
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/bases/BlockBase.java:12]: Missing space between class initialization and method

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Review by Desoroxxx — APPROVED** [review]

**Pull request merged**
